kubernetes/kserve/kserve_wrapper/TorchserveModel.py

```diff
@@ -53,72 +55,103 @@ def __init__(self, name, inference_address, management_address, model_dir):
         self.explainer_host = self.predictor_host
         logging.info("kfmodel Explain URL set to %s", self.explainer_host)
 
-    async def predict(self, request: Dict) -> Dict:
+    async def predict(self, payload: Union[Dict, InferRequest], headers: Dict) -> Dict:
         """The predict method is called when we hit the inference endpoint and handles
         the inference request and response from the Torchserve side and passes it on
         to the KServe side.
 
         Args:
-            request (Dict): Input request from the http client side.
+            payload (Dict|InferRequest): Input payload from the http client side.
+            headers (Dict): Request headers.
 
         Raises:
             NotImplementedError: If the predictor host on the KServe side is not
                                  available.
 
-            tornado.web.HTTPError: If there is a bad response from the http client.
+            HTTPStatusError: If there is a bad response from the http client.
 
         Returns:
             Dict: The Response from the input from the inference endpoint.
         """
         if not self.predictor_host:
             raise NotImplementedError
-        logging.debug("kfmodel predict request is %s", json.dumps(request))
+        if isinstance(payload, InferRequest):
+            payload = payload.to_rest()
+        data = orjson.dumps(payload)
```

Comment:
Is there a reason for using orjson instead of json?
Also is this in python by default or do you need to pip install?